### PR TITLE
Issue 2 - Add initial support for Markdown (update)

### PR DESCRIPTION
**Pull request by kiennguyenchi**
I updated some code to handle support feature for Markdown page input.
1/ In txt2html.js, I create an else-if in txt2html function to process the .md file. It will pass the text of .md file to an imported function generateFromMd() for converting purpose.
2/In generator.js, I create a function named generateFromMd() to convert .md file into .html file. Also, I add a convertToH1() function to convert all heading 1 into tag h1 in html. For other features in .md files have not been supported yet.
Basically, I haven't changed any code structure in your previous code. Please review my pull request carefully. Thank you!

### CODE REVIEW

**Comment by tcvan0707** [comment]
Thank you very much.
